benefits/templates/200-user-error.html

```diff
@@ -1,21 +1,25 @@
-{% extends "error.html" %}
+{% extends "error-base.html" %}
 {% load i18n %}
 
 {% block title %}
   {% translate "Start over" %}
 {% endblock title %}
 
-{% block headline %}
+{% block icon %}
+  {% include "core/includes/icon.html" with name="sadbus" %}
```

Comment:
I love this refactor to have all error pages inherit from `error-base.html` :heart_eyes: 

I think this icon include could be moved there as the "default" icon. Which means it wouldn't have to be repeated on all these regular error pages.

That doesn't prevent us from overriding it on error pages that do need a custom icon.

Reply:
Ooooh yes!

Reply:
Default sad bus.

Reply:
Done